client/src/features/settings/Settings.tsx

```diff
@@ -1,7 +1,225 @@
+											<span className={`p-input-icon-right w-full ${touched.password && errors.password ? 'border-red-500' : 'border-gray-300'}`}>
+												<Password
+													id="password"
+													name="password"
+													value={values.password}
+													onChange={handleChange}
+													toggleMask
+													className="w-full"
+													inputClassName="w-full"
+													feedback={false}
+													placeholder="Enter password"
+													pt={{
+														input: { className: 'w-full' },
+														// showIcon: { className: 'h-full flex items-center' },
+														// hideIcon: { className: 'h-full flex items-center' }
+													}}
+												/>
+											</span>
+											{touched.password && errors.password && (
+												<small className="text-red-600">{errors.password}</small>
+											)}
+										</div>
+									</div>
+
+									<div className="flex-1">
+										<div className="flex flex-col gap-2">
+											<label htmlFor="confirmPassword">Confirm Password</label>
+											<span className={`p-input-icon-right w-full ${touched.confirmPassword && errors.confirmPassword ? 'border-red-500' : 'border-gray-300'}`}>
+												<Password
+													id="confirmPassword"
+													name="confirmPassword"
+													value={values.confirmPassword}
+													onChange={handleChange}
+													toggleMask
+													className="w-full"
+													inputClassName="w-full"
+													feedback={false}
+													placeholder="Confirm password"
+													pt={{
+														input: { className: 'w-full' },
+														// showIcon: { className: 'h-full flex items-center' },
+														// hideIcon: { className: 'h-full flex items-center' }
+													}}
+												/>
+											</span>
+											{touched.confirmPassword && errors.confirmPassword && (
+												<small className="text-red-600">{errors.confirmPassword}</small>
+											)}
+										</div>
+									</div>
+
+									<div className="flex-1">
+										<div className="flex flex-col gap-2">
+											<label htmlFor="language">Language</label>
+											<Dropdown
+												id="language"
+												name="language"
+												value={values.language}
+												options={languages}
+												onChange={(e) => setFieldValue('language', e.value)}
+												className={`w-full ${touched.language && errors.language ? 'border-red-500' : 'border-gray-300'}`}
+												placeholder="Select language"
+											/>
+											{touched.language && errors.language && (
+												<small className="text-red-600">{errors.language}</small>
+											)}
+										</div>
+									</div>
+								</div>
+							</div>
+
+							<div className="mt-4">
+								<ActionButtons
+									handleSubmit={submitForm}
+									handleResetForm={resetForm}
+									handleCloseDialog={() => { }} // Empty function since this is not in a dialog
```

Comment:
Hum. I see. In this case, I think I have to change the ActionButtons component to make this optional.